### PR TITLE
Stabilize three e2e tests

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: tests/e2e/reports/report-date-filter.wdio-spec.js [hunk 1]
```diff
@@ -99,9 +99,9 @@ describe('Filters reports', () => {
     await reportsTab.filterByDate(moment('05/16/2016','MM/DD/YYYY'), moment('05/17/2016','MM/DD/YYYY'));
     await (await reportsTab.firstReport()).waitForDisplayed();
 
-    expect(await reportsTab.allReports().length).to.equal(2);
-    expect(await reportsTab.reportsByUUID(savedUuids[1]).length).to.equal(1);
-    expect(await reportsTab.reportsByUUID(savedUuids[3]).length).to.equal(1);
+    expect(await (await reportsTab.allReports()).length).to.equal(2);
```
Comment (m5r) [inline tests/e2e/reports/report-date-filter.wdio-spec.js:102]: I was expecting to be able to remove one of the two `await` like so
```ts
expect((await reportsTab.allReports()).length).to.equal(2);
```

But I've tried reverting this change and it's still returning `2`. Do we really need to include these changes?

**Review by m5r — CHANGES_REQUESTED** [review]
Nicely done! I've left one question inline

File: tests/e2e/reports/report-date-filter.wdio-spec.js (hunk 1 above)
Comment (dianabarsan) [reply tests/e2e/reports/report-date-filter.wdio-spec.js:102]: These were failing for me locally, so I think this is a race condition that sometimes works.

Comment (m5r) [reply tests/e2e/reports/report-date-filter.wdio-spec.js:102]: Good job catching it!

**Review by m5r — APPROVED** [review]

**Pull request merged**